### PR TITLE
Fixes some IE11 visual bugs

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:

* Fixes missing gap between icon and text in some Infobox rendering in IE11
* Hides native show/hide icon in IE11 because we have our own from components

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-651388](https://oktainc.atlassian.net/browse/OKTA-651388)

### Screenshot/Video:
Password field no longer shows duplicate show/hide icon:
20230925_20_16_36_internet explorer_11

Alert icon has proper spacing:
20230925_20_16_06_internet explorer_11

### CODE REVIEW

File: src/v3/src/util/mergeThemes.ts [hunk 1]
```diff
@@ -73,6 +73,12 @@ export const mergeThemes = (
                   // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
                   ...resolve(config.styleOverrides?.icon, options),
                 }),
+                input: (options: Record<string, unknown>) => ({
+                  // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
+                  ...resolve(prev.components?.[component]?.styleOverrides?.input, options),
+                  // @ts-expect-error FIXME CSSInterpolation may not be CSSObject
+                  ...resolve(config.styleOverrides?.input, options),
+                }),
```
Comment (leemchale-okta) [inline src/v3/src/util/mergeThemes.ts:81]: what does this do?

**Review by leemchale-okta — APPROVED** [review]

File: src/v3/src/util/mergeThemes.ts (hunk 1 above)
Comment (shawyu-okta) [reply src/v3/src/util/mergeThemes.ts:81]: This utility merges the MUI theme configuration -- right now it's not fully generalized for the properties within `styleOverrides` so I'm just duplicating the inner logic for each one we need to handle. @lesterchoi-okta is updating the utility to eventually handle this better but for expedience we're just doing this right now.

File: scripts/e2e.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ setup_service java 1.8.222
 setup_service google-chrome-stable 89.0.4389.72-1
 
 export TEST_SUITE_TYPE="junit"
-export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2"
```
Comment (shawyu-okta) [inline scripts/e2e.sh:11]: Bacon scripts zip whatever directory is listed in this env variable and saves it to `TestResults.zip`.

Comment (shawyu-okta) [reply scripts/e2e.sh:11]: This ensures we capture `/build2/reports/junit` still but also include the `/build2/screenshots` directory that contains the screenshots captured in these test runs.

**Pull request merged**